### ISSUE TITLE
Make sure GA is properly set up and respects user's DNT preference

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozillafestival.org#806]

**Comment by mmmavis** [comment]
**BLOCKED**

I'm still unable to get access to the GA Dashboard... Multiple tickets have been filed and we have tried reaching out to admins but still haven't heard back from them.

**Comment by mmmavis** [comment]
From yesterday's meeting we have decided to create a new GA property that MoFo owns. I'll create a DevOps ticket for it.

**Comment by gideonthomas** [comment]
Property created, @mmmavis let me know if you aren't able to access it.

Tracking ID: `UA-87658599-15`

**Comment by marcwalsh** [comment]
@gideonthomas @mmmavis - would it be possible to use this GA property to authenticate Google Search Console as well. I created a separate ticket here for it #1070 - but if we have GA access we can just use that. 

(Again no rush with this, it's still a nice to have if either of you have time.)

**Comment by gideonthomas** [comment]
@marcwalsh yeah, I can do it right now

**Comment by gideonthomas** [comment]
Looks like GA needs to be added to the site before I can authenticate Search Console. Once this ticket is done, I can do that.

@mmmavis, just so that you only read the last comment, tracking ID is: `UA-87658599-15`

**Comment by mmmavis** [comment]
@gideonthomas you are unblocked for the Search Console ticket! Please don't close this ticket yet as I still need to take of our the DNT part.

**Comment by gideonthomas** [comment]
ok, thanks @mmmavis!
